identity-account-storage/src/stronghold/mod.rs

```diff
@@ -4,7 +4,8 @@
 mod context;
 mod error;
 mod hint;
-mod records;
+// TODO: Remove or refactor for key deletion.
+//mod records;
 mod snapshot;
```

Comment:
TODO

Reply:
Should be handled with https://github.com/iotaledger/identity.rs/issues/757, where we might want to use the `Records` or go with another solution to keep track of keys. There we can make a final decision on whether to remove or use it.